tools/release

```diff
@@ -51,17 +51,19 @@ if 'upload' not in sys.argv:
     sys.exit(0)
 
 # Register with the Python Package Index (PyPI)
-print( 'Registering with PyPI...')
+print( 'Not Registering with PyPI, registering with setup.py is insecure as communication is not encrypted')
 cd(ipdir)
-sh('./setup.py register')
+# sh('./setup.py register')
 
 # Upload all files
-sh(sdists + ' upload')
+sh(sdists)
 for py in ('2.7', '3.4'):
-    sh('python%s setupegg.py bdist_wheel upload' % py)
+    sh('python%s setupegg.py bdist_wheel' % py)
+
+print('Will not upload with setuptools as upload connection is insecure. Please use `twine upload dist/*` to upload the files to PyPI')
```

Comment:
I was thinking of just putting `sh('twine upload dist/*')` in there. No need to work out how to use it as a library. ;-)